eng/common/docgeneration/templates/matthews/styles/main.js

```diff
@@ -4,9 +4,9 @@ containers.removeClass("container");
 containers.addClass("container-fluid");
 
 WINDOW_CONTENTS = window.location.href.split('/')
-var SELECTED_LANGUAGE = ''
-var PACKAGE_REGEX = ''
-var PACKAGE_REPLACEMENT = ''
+var SELECTED_LANGUAGE = 'java'
```

Comment:
Looks like you left in your test language. I believe that needs to be reverted from eng/common.